2024/json-ld-wg/index.html

```diff
@@ -144,9 +144,26 @@ <h1 id="title"><i class=todo>DRAFT</i> JSON-LD Working Group Charter</h1>
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
         <p>
-          JSON-LD (JavaScript Object Notation for Linked Data) is a method of encoding linked data using JSON.
-          The encoding is used used mostly for search engine optimization activities but also for applications such as
-          biomedical informatics, and representing provenance information.
+          JSON-LD (JavaScript Object Notation for Linked Data) is a widely used method of encoding Linked Data using JSON.
+          Use of JSON-LD has grown and expanded over the last decade (since it's first publication as a
+          <a href="https://www.w3.org/TR/json-ld-api1/">W3C Recommendation in 2014</a>).
+          JSON-LD remains a popular choice for ecoding <abbr title="Seach Engine Optimization">SEO</abbr>-focused
+          structured data in Web pages. JSON-LD has also become a foundational format for social media (ActivityStreams),
+          Internet of Things data (Web of Things Description), Verifiable Credentials, and Software Bill of Materials (SPDX).
+        </p>
+        <p>This growth in the applied use of JSON-LD has resulted in community interest in additional expressions of
+          JSON-LD's underlying Linked Data expression into other formats. YAML-LD has been created by the
```

Comment:
"expression" used twice, making the sentence cumbersome.

```suggestion
        <p>This growth in the applied use of JSON-LD has resulted in community interest in additional expressions of
          JSON-LD's underlying Linked Data structure into other formats. YAML-LD has been created by the
```

Reply:
Maybe just keep the group title "JSON-LD Working Group" and then expand in the group description, and/or in a subtitle, that it is for JSON and related formats.